decimal.go

```diff
@@ -1684,3 +1683,15 @@ func (d Decimal) Tan() Decimal {
 	}
 	return y
 }
+
+// Copy makes instance of d with same value and different pointer.
+//
```

Comment:
nitpick: I think this line is unnecessary :D

Reply:
I agree